app/controllers/project_approvals_controller.rb

```diff
@@ -1,25 +1,21 @@
 # Allows a project to be approved
 class ProjectApprovalsController < ModerationController
-  expose(:project_approval, scope: -> { pending_project.project_status_changes },
-                            model: :project_status_change)
-  expose(:pending_project, scope: -> { Project.pending }, model: :project)
+  expose(:approval, scope: -> { project.project_status_changes },
```

Comment:
This rename should have been done in a different PR but I got tripped up in it due to weird naming issues as I was making the feature work. Can either do it separately or back it out now that I know how to fix the weird naming issues.